src/Braket.jl

```diff
@@ -134,6 +139,72 @@ end
 Base.copy(fp::FreeParameter) = fp
 Base.show(io::IO, fp::FreeParameter) = print(io, string(fp.name))
 
+"""
+    FreeParameterExpression
+    FreeParameterExpression(expr::Union{FreeParameterExpression, Number, Symbolics.Num, String})
+
+Struct representing a Free Parameter expression, which can be used in symbolic computations.
```

Comment:
Docstring here should link to the one for `FreeParameter`

Reply:
Added!